viewer/apiSessions.js

```diff
@@ -1032,9 +1040,9 @@ module.exports = (Config, Db, decode, internals, molochparser, Pcap, ViewerUtils
         return nextCb(null);
       }
 
-      let node = (Config.get('multiES', false) && session._node) ? session._node : undefined;
+      let escluster = (Config.get('multiES', false) && session._source.escluster) ? session._source.escluster : undefined;
```

Comment:
I have added to both **_source** and to the **top level**. Need to add the "escluser" field to __source so that it can be used to fetch session details.